flux/stdlib/influxdata/influxdb/to.go

```diff
@@ -217,12 +300,12 @@ func (t *ToTransformation) Process(id execute.DatasetID, tbl flux.Table) error {
 			measurementName := ""
 			fields := make(models.Fields)
 			var pointTime time.Time
-			kv = kv[0:]
+			kv = kv[:0]
```

Comment:
This was obviously bad but happened to pretty often work properly, probably due to stable-ish sorting somewhere. The desired behaviour is truncation, not a no-op!!

Reply:
nice catch